### ISSUE TITLE
Query strings break HTML environment

### ISSUE THREAD

**Issue by Potato42** [Khan/live-editor#626]
Attempting to navigate to a query string does not work in the HTML environment, and worse it actually breaks the environment so that even the "Reload" button won't fix it.

An example of this problem can be found [here](https://www.khanacademy.org/computer-programming/iter-tests/6514431734185984).  You need to press "Reload" once for the program to start (an unrelated problem, not really sure why that's happening).  After that, click on just about anything in the Jasmine interface.  For example, if you click on "iter.Iterator", Jasmine tries to add the query string "spec=iter module iter.Iterator" to the url, but it breaks.

If it was possible to prevent, ignore, or ideally handle navigation to query strings properly, it would be nice.

**Comment by kevinbarabash** [comment]
@Potato42 I opened `/computer-programming/iter-tests/6514431734185984` and clicked reload and nothing happened.  I also tried clicked on `iter.Iterator` and nothing happened.  Do I have to copy/paste this into a new project?  Could you create a minimal example with the bug?

**Comment by Potato42** [comment]
How did you click on iter.Iterator if you couldn't get it to load in the first place?  Here's a minimal example: https://www.khanacademy.org/computer-programming/query-string/4816496181182464

**Comment by kevinbarabash** [comment]
@Potato42 the code loaded in the editor, but nothing happened on the canvas.  Thanks for the minimal example.  I don't think there are any lessons on KA that use query params so this probably won't get looked at until we decided to reinvest in CS.  Good to have the issues tracked.

**Comment by Potato42** [comment]
Yeah, there are no lessons that use them and it is unlikely that anyone would want to use them in the KA environment.  The only real problem is libraries like jasmine's html viewer that do.
